<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forma_Pago</class>
 <widget class="QDialog" name="Forma_Pago">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icono.png</normaloff>icono.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Forma_Pago{
	border-image: url(:/Img/F2.jfif);}</string>
  </property>
  <widget class="QDialogButtonBox" name="Entrar_Pago_buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>200</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-width: 0px;
border-radius: 8px;
background: rgba(0, 0, 0,80);}

QPushButton:hover{
	background-color: rgb(154,158,218);
    border-radius: 9px;
	color: black;
    selection-color: white;  
    padding: 2px 4px;

}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="Datos" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>411</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Datos{ 
background: rgba(0, 0, 0, 80);
 border-radius: 9px;}

QLineEdit{
background-color:transparent;
border:none;
border-bottom: 1px solid;
border-color: rgb(56, 143, 147);}</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>CVC: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Tipo de tarjeta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tipo_comboBox">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="Numero_Tarjeta_lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>319</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maxLength">
     <number>32766</number>
    </property>
    <property name="placeholderText">
     <string>NÃºmero de Tarjeta</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>230</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="Caducidad_lineEdit">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Fecha de Vencimiento</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Caducidad_lineEdit_2">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Fecha de Vencimiento</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DATOS BANCARIOS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
